client/jetpack-connect/controller.js

```diff
@@ -293,3 +294,29 @@ export function plansSelection( context, next ) {
 	);
 	next();
 }
+
+/**
+ * Checks for a locale fragment at the end of context.path
+ * and switches to that locale if the user is logged out.
+ * If the user is logged in we remove the fragment and defer to the user's settings.
+ *
+ * @param {Object} context -- Middleware context
+ * @param {Function} next -- Call next middleware in chain
+ * @returns {Undefined} next()
```

Comment:
Nitpick: This return value description is not quite true :)

It will only be something from `next()` when it returns from the line 312. When it reaches the line 321 and ends, it will just return `undefined` and that is not something from `next()`.

Just wanted to mention it so please don't bother open another PR for fixing it. Super duper minor :)